pkg/configuration/configuration.go

```diff
@@ -178,7 +178,7 @@ func (r AuthConfig) AuthClientConfigRaw() string {
 }
 
 func (r AuthConfig) AuthClientPublicKeysURL() string {
-	return commonconfig.GetString(r.c.AuthClientPublicKeysURL, "https://sso.devsandbox.dev/auth/realms/sandbox-dev/protocol/openid-connect/certs")
```

Comment:
Is it a new keycloak thing? Or specific to our dev sso configuration?
We do rely on `auth/realms` path in a few places which is consistent with our production sso.redhat.com.
If we have a different path in dev sso then it won't work OOTB.
See https://github.com/search?q=repo%3Acodeready-toolchain%2Fregistration-service+%22auth%2Frealms%22&type=code